<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Roblox Studio Discord RPC</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../assets/assets.qrc">
    <normaloff>:/app.png</normaloff>:/app.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabs">
      <widget class="QWidget" name="homeTab">
       <attribute name="icon">
        <iconset resource="../assets/assets.qrc">
         <normaloff>:/icons/home.png</normaloff>:/icons/home.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Home</string>
       </attribute>
       <attribute name="toolTip">
        <string>Home page</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="statusBox">
          <property name="title">
           <string>Status</string>
          </property>
          <layout class="QFormLayout" name="formLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="dscFound">
             <property name="text">
              <string>Discord:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="rbxstudioFound">
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text">
              <string>Roblox Studio:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="FindStatusDisplay" name="dscStatus" native="true"/>
           </item>
           <item row="1" column="1">
            <widget class="FindStatusDisplay" name="rbxstudioStatus" native="true"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="settingsTab">
       <attribute name="icon">
        <iconset resource="../assets/assets.qrc">
         <normaloff>:/icons/gear.png</normaloff>:/icons/gear.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="launchOnLoginLabel">
          <property name="toolTip">
           <string>The service starts when you log-in your computer, and will only show rich presence when it finds that Roblox Studio is opened.</string>
          </property>
          <property name="text">
           <string>Launch on log-in</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="launchOnLoginCheck">
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="closeExitLabel">
          <property name="text">
           <string>Exit service on window close</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="closeExitCheck">
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="abtTab">
       <attribute name="icon">
        <iconset resource="../assets/assets.qrc">
         <normaloff>:/icons/information.png</normaloff>:/icons/information.png</iconset>
       </attribute>
       <attribute name="title">
        <string>About</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="progName">
          <property name="font">
           <font>
            <pointsize>15</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Roblox Studio Discord RPC</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="Line" name="sep">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="vanillaCredit">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;https://github.com/Elttob/Vanilla&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#007af4;&quot;&gt;Vanilla&lt;/span&gt;&lt;/a&gt; icons by &lt;a href=&quot;https://github.com/Elttob/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#007af4;&quot;&gt;Elttob&lt;/span&gt;&lt;/a&gt; for rich presence icons&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="progDesc">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>This program connects to a Roblox Studio plugin, to display Discord custom rich presence for Roblox Studio.</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="copyright">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;By &lt;a href=&quot;https://github.com/MatusGuy&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#007af4;&quot;&gt;MatusGuy&lt;/span&gt;&lt;/a&gt; &amp;amp; Localtings Â© 2022-2023&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <layout class="QHBoxLayout" name="btnLnkLayout">
          <item>
           <widget class="HyperlinkButton" name="programRepo">
            <property name="text">
             <string>Program GitHub repository</string>
            </property>
            <property name="destination" stdset="0">
             <url>
              <string>https://github.com/RobloxStudio-DiscordRPC/Plugin</string>
             </url>
            </property>
           </widget>
          </item>
          <item>
           <widget class="HyperlinkButton" name="pluginRepo">
            <property name="text">
             <string>Roblox Studio plugin GitHub repository</string>
            </property>
            <property name="destination" stdset="0">
             <url>
              <string>https://github.com/RobloxStudio-DiscordRPC/Plugin</string>
             </url>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="5" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="btnLayout">
      <item>
       <spacer name="spacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="exitBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FindStatusDisplay</class>
   <extends>QWidget</extends>
   <header>findstatusdisplay.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>HyperlinkButton</class>
   <extends>QPushButton</extends>
   <header>hyperlinkbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../assets/assets.qrc"/>
 </resources>
 <connections/>
</ui>
